lib/healthcheck/api_tokens.rb

```diff
@@ -0,0 +1,47 @@
+module Healthcheck
+  class ApiTokens
+    WARNING_THRESHOLD = 1.year.to_i
+    CRITICAL_THRESHOLD = 1.month.to_i
+
+    QUERY = <<-SQL.freeze
+      SELECT users.name, oauth_applications.name FROM oauth_access_tokens
+      INNER JOIN users ON users.id = resource_owner_id
+      INNER JOIN oauth_applications ON oauth_applications.id = application_id
+      WHERE oauth_access_tokens.revoked_at IS NULL
+      AND users.api_user = TRUE
+      AND (oauth_access_tokens.expires_in -
+          (unix_timestamp(now()) -
+           unix_timestamp(oauth_access_tokens.created_at))) < %<threshold>s
+    SQL
```

Comment:
Do you know if this sql is ok for both mysql and postgres? I believe (and would like to be wrong) that this app supports both databases.

Reply:
@tijmenb thinks we can drop Postgres support - the tech debt card doesn't mention a particular use-case we're aiming for, aside from migrating fully to one database.

https://trello.com/c/Zw8xD510/61-signon-supports-both-postgresql-and-mysql

Reply:
Great - be good to get rid of one!